Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -268,7 +269,15 @@ def retrieve_and_display_info_and_open_pr(formula_or_cask, name, repositories, a
     EOS
 
     return unless args.open_pr?
-    return unless new_version
+
+    unless new_version
+      puts <<~EOS
+        #{title_name} was not bumped because either:
+         - it has no new versions, or
+         - its Repology version is not the latest release on GitHub.
+      EOS
+      return
+    end
```

Comment:
How about this? This way, if the `livecheck` version is greater than the current version but lesser than the Repology version, the user isn't confused as to why `bump` decided to go for the `livecheck` version.
```suggestion
    if repology_latest > current_version && 
       repology_latest > livecheck_latest &&
       livecheck_strategy == "GithubLatest"
      puts "#{title_name} was not bumped to the Repology version because that " \
           "version is not the latest release on GitHub."
    end
    
    return unless new_version
```

Reply:
This sort of duplicates the logic from the computation of `new_version`. But sure, this more intentionally shows the user what why the tool made the decision. Let's go for it.